openmetadata-service/src/main/resources/json/data/testConnections/database/snowflake.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
How we are testing the steps defined in these schemas and the steps we are running are the same?

For example, for snowflake we are running also `GetDatabases` that does not appear here.